packages/webdriverio/.eslintrc.js

```diff
@@ -1,15 +1,7 @@
 module.exports = {
   rules: {
-    '@typescript-eslint/no-empty-function': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
I don't think we should disable this rule. Using `any` defeats the purpose of TypeScript. Can we change this to a warning instead?

```suggestion
    '@typescript-eslint/no-explicit-any': 'warn',
```

Reply:
I could just remove this from eslint and add `// @ts-ignore` where I explicitly use `any` type which is only in test files.

Reply:
If it's only in test files, then I'm strongly against turning this rule off globally. Just use an `override` instead:

```json
{
  "overrides": [
    {
      "files": ["*.test.ts", "*.test.tsx"],
      "rules": {
        "@typescript-eslint/no-explicit-any": "off"
      }
    }
  ]
}
```

Reply:
I usually have something like this in my `eslintrc`:

```js
{
  "overrides": [
    {
      "files": ["*.test.ts", "*.test.tsx"],
      "rules": {
        // Tests can use `any`.
        "@typescript-eslint/no-explicit-any": "off",
        // Noop mocks, etc.
        "@typescript-eslint/no-empty-function": "off",
        // Tests can disable the TS compiler.
        "@typescript-eslint/ban-ts-comment": "off"
      }
    }
  ]
}
```